src/yardstick/cli/label.py

```diff
@@ -159,10 +159,10 @@ def remove_label(
     label_ids: list[str],
 ):
     label_entries = store.labels.load_all()
-    old_len = len(label_entries)
-    label_entries = [l for l in label_entries if l.ID not in label_ids]
-    store.labels.overwrite_all(label_entries)
-    logging.info(f"removed {old_len-len(label_entries)} labels")
+    to_delete = [l for l in label_entries if l.ID in label_ids]
+    to_keep = [l for l in label_entries if l.ID not in label_ids]
```

Comment:
it would be nice if we could enhance this capability so that we didn't need to load all of the labels (which takes a while and isn't really necessary for a delete operation). It might be time to enhance the `store.label` with a `delete` function which can be given a set of IDs to delete, which could do `glob` operations to find it. This would also mean simplifying the current implementation of the `store_filename_by_entry`.